frontend/src/scenes/notebooks/Notebook/Notebook.scss

```diff
@@ -36,12 +36,40 @@
                 height: 0;
             }
 
-            > ul {
+            > ol {
```

Comment:
this makes me wonder about people's expectations when editing...

We'll now have notebooks that do support this well and dashboard text cards that don't support this at all...

Not blocking here and I guess other work resolves this distinction